package/scripts/params.py

```diff
@@ -73,15 +72,18 @@
 else:
     hdfs_namespace = map_cdap_site['hdfs.namespace']
 
+cdap_security_enabled = config['configurations']['cdap-site']['security.enabled']
+
 # Kerberos stuff
+kerberos_enabled = config['configurations']['cluster-env']['security_enabled']
```

Comment:
on second thought... assuming this is just fetches ``security.enabled`` from cdap-site.xml, my real question is just that i don't see where ``cdap_security_enabled`` (on line 75) is used...